pkg/engine/template_generator.go

```diff
@@ -150,6 +150,9 @@ func (t *TemplateGenerator) getTemplateFuncMap(vm *api.APIModel) template.FuncMa
 		"GetDataDisks": func(p *api.Properties) string {
 			return getDataDisks(p.VMProfile)
 		},
+		"IsSNPEnabled": func() bool {
```

Comment:
as before, rename to IsSecureBootEnabled ?

Reply:
Resolved in the new iteration